packages/react-components/src/molecules/Accordion.tsx

```diff
@@ -79,39 +108,54 @@ type AccordionProps = {
     href: string;
     hrefText: string;
   }[];
+  info?: {
+    text: string;
+    href: string;
+    hrefText: string;
+  };
 };
 
-const Accordion: FC<AccordionProps> = ({ items }) => {
+const Accordion: FC<AccordionProps> = ({ items, info }) => {
   const [opened, setOpened] = useState<number | undefined>();
   return (
     <Card accent="neutral200" padding={false}>
       <div css={containerStyles}>
         {items.map(({ icon, title, description, href, hrefText }, index) => (
           <div key={`accordion-${index}`} css={itemStyles}>
             <button
-              css={[expandButtonStyles, headerStyles]}
+              css={[buttonStyles, headerStyles]}
               onClick={() => setOpened(opened === index ? undefined : index)}
             >
               <div css={iconStyles}>{icon}</div>
               <Headline5>{title}</Headline5>
-
               <div css={[iconStyles, index === opened && iconOpenStyles]}>
                 {chevronDownIcon}
               </div>
             </button>
-            <div
-              css={[bodyStyles, hiddenStyles, index === opened && openStyles]}
-            >
-              <Paragraph accent={'lead'}>{description}</Paragraph>
-              <div css={{ width: 'fit-content' }}>
-                <Link buttonStyle small primary href={href}>
-                  {hrefText} {!isInternalLink(href)[0] && externalLinkIcon}
-                </Link>
+            <div css={[hiddenStyles, index === opened && openStyles]}>
+              <div css={bodyStyles}>
+                <Paragraph accent={'lead'}>{description}</Paragraph>
+                <div css={{ width: 'fit-content' }}>
```

Comment:
Is it ok to use "inline" css directly in the html tag? I assume it would be fine for this case as it doesn't receive any variables + it's only one css prop.

Reply:
I think its okay to do for like one css rule but mostly we prefer to separate it where possible.